_tutorials/102_march/step-09/102_march_step-09-desc.html

```diff
@@ -1,9 +1,11 @@
 <div>
-    <p id="tutorialGoal">You should be prepared now to encode the note values of the final two measures of the example.</p>
+    <p id="tutorialGoal">Step nine: You should now be ready to encode the note values of the final two measures of the example.</p>
 
-    <p>Ignore the additional elements, such as dynamics, slurs and accents at this point as they will be adressed in the next steps separately.</p>
+    <p>Ignore the additional elements, such as dynamics, slurs and accents at this point as they will be adressed separately in the next steps.</p>
 
     <p>The pause is encoded with a <code>&lt;rest&gt;</code> element having a <code>@dur</code> attribute similar to the <code>&lt;note&gt;</code> element.</p>
 
-    <p id="tutorialTask">Please encode the final two measures (<code>n=3</code> & <code>n=4</code>) of the example in the editor below. Add <code>&lt;measure&gt;</code>, <code>&lt;staff&gt;</code> and <code>&lt;layer&gt;</code> elements as well as the corresponding <code>&lt;note&gt;</code>, <code>&lt;beam&gt;</code> or <code>&lt;rest&gt;</code> elements. This is the most ambitious step of the tutorial. If you feel lost or unsure at any time, try to follow the hints that are displayed next to the editor box.</p>
+    <p id="tutorialTask">Please encode the final two measures (<code>n=3</code> & <code>n=4</code>) of the example in the editor below. Add <code>&lt;measure&gt;</code>,
+    <code>&lt;staff&gt;</code> and <code>&lt;layer&gt;</code> elements as well as the corresponding <code>&lt;note&gt;</code>, <code>&lt;beam&gt;</code> or <code>&lt;rest&gt;</code> elements.
+    This is the most ambitious step of the tutorial. If you feel lost or unsure at any time, try to follow the hints that are displayed next to the editor box.</p>
```

Comment:
Maybe better: ... most ambitious step of **this** tutorial ...

Reply:
Yep.